app/views/histories/new.html.haml

```diff
@@ -0,0 +1,5 @@
+%h1 New history
```

Comment:
I noticed I was not able to enter anything on this form, how do you envision it working?

Reply:
I believe I did mean to remove this when I created the scaffold for the history. I can modify that.

Reply:
I see, thanks for clarifying.